packages/commonwealth/server/models/membership.ts

```diff
@@ -0,0 +1,44 @@
+import type * as Sequelize from 'sequelize';
+import { DataTypes } from 'sequelize';
+import { ModelInstance, ModelStatic } from './types';
+import { GroupAttributes } from './group';
+
+export type MembershipAttributes = {
+  group_id: number;
+  address_id: number;
+  last_checked: Date;
+
+  // associations
+  Group?: GroupAttributes;
+};
+
+export type MembershipInstance = ModelInstance<MembershipAttributes>;
+export type MembershipModelStatic = ModelStatic<MembershipInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): MembershipModelStatic => {
+  const Membership = <MembershipModelStatic>sequelize.define(
+    'Membership',
+    {
+      group_id: { type: dataTypes.INTEGER, allowNull: false },
```

Comment:
Should we store the value of the result (yes / no)? or just when checked?

Reply:
Forgot that. Will add.